src/common/snippets/src/lowered/pass/validate_unified_loops.cpp

```diff
@@ -88,6 +81,45 @@ bool ValidateUnifiedLoops::run(LinearIR& linear_ir) {
         OPENVINO_ASSERT(unique_dimensions.size() <= 1,
                         "Loop ports have incompatible dimensions, by which the loop iterates");
     }
+}
+
+void ValidateUnifiedLoops::validate_loop_port_presence(const LinearIR& linear_ir) {
+    auto validate_loop_port = [](const ExpressionPort& expr_port, const LoopInfoPtr& loop_info, size_t loop_id) {
+#define VALIDATE_LOOP_PORT(check, txt) \
+    OPENVINO_ASSERT(check, "Expression port with idx ", expr_port.get_index(), " with node ", expr_port.get_expr()->get_node()->get_friendly_name(), txt);
```

Comment:
This defined is used only twice, and I think it would be more readable if we just repeat this assert twice.

Reply:
Okey, done